### PR TITLE
Update macOS installation docs

### PR DESCRIPTION
**Pull request by cmluciano**

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

File: site/docs/install.md [hunk 1]
```diff
@@ -67,12 +67,19 @@ package:
...
+
+        ```
+        fpath[1,0]=~/.zsh/completion/
+        mkdir -p ~/.zsh/completion/
+        cp scripts/zsh_completion/_bazel ~/.zsh/completion
+        ```
+    2. macOS
+
+        ```
+        cp /usr/local/Cellar/bazel/0.5.1/share/zsh/site-functions/_bazel  ./completion/.
```
Comment (damienmg) [inline site/docs/install.md]: What is ./completion/.?
Why do you need a special step on macOS? Isn't homebrew supposed to install that correctly?

Also why is the step different from Linux? I mean the origin should be the git repository probably.

Comment (cmluciano) [inline site/docs/install.md]: That's a good point. Perhaps I should patch the brew package to ensure that this happens.

**Comment by damienmg** [comment]
Closing this PR (stalled)

**Pull request closed without merging**